types/index.d.ts

```diff
@@ -0,0 +1,7 @@
+import type { ESLint, Linter } from "eslint"
+
+export declare const configs: { recommended: Linter.FlatConfig }
```

Comment:
I believe this should be `LegacyConfig`, since the `configs` export is [lib/configs](https://github.com/eslint-community/eslint-plugin-eslint-comments/blob/60c28f9a5530e29991f7a388d28ad0aef5af4529/index.js#L5) which contains legacy string-array `plugins`: https://github.com/eslint-community/eslint-plugin-eslint-comments/blob/60c28f9a5530e29991f7a388d28ad0aef5af4529/lib/configs/recommended.js#L5